src/crimson/os/alienstore/alien_store.h

```diff
@@ -21,6 +21,22 @@ class Transaction;
 namespace crimson::os {
 class AlienStore final : public FuturizedStore {
 public:
+  class AlienOmapIterator : public OmapIterator {
+    ObjectMap::ObjectMapIterator oiter;
```

Comment:
could you put the private members at the end of the class? see https://google.github.io/styleguide/cppguide.html#Declaration_Order